spec/support/fixtures.rb

```diff
@@ -1585,7 +1585,7 @@ def handle(req, res)
   end
 
   class CustomFromAccept < Hanami::Action
-    config.add_format custom: "application/custom"
```

Comment:
@timriley If you don't like this removal, we can still reintroduce `#add_format`.